packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.class.js

```diff
@@ -0,0 +1,49 @@
+import get from 'lodash/get';
+
+import {
+  BACKUP_STATE_DISABLED,
+  BACKUP_OFFER_LEGACY,
+  BACKUP_OFFER_ADVANCED,
+  BACKUP_OFFER_PREMIUM,
+} from './backup.constants';
+
+export default class Backup {
+  constructor(backup) {
+    Object.assign(this, backup);
+    this.initializeBackup();
+  }
+
+  initializeBackup() {
+    this.backupSizeInReport = this.backupSizeInReport
```

Comment:
Maybe this could be done with destructuring in the constructor and something like 
```suggestion
    this.backupSizeInReport = backupSizeInReport || false
```

Reply:
This is done :)